features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/subscription/list.jag

```diff
@@ -68,6 +68,28 @@ var getAllSubscriptionsOfApplicationById = function (username, selectedAppId, st
     }
 };
 
+var getAllScopesOfApplicationSubscription = function (username, selectedAppId)  {
+    var result,
+    log = new Log(),
+    store = jagg.module("manager"). getAPIConsumerObj();
+    try {
+	    result = store.getScopesForApplicationSubscription(username , selectedAppId);
+        if (log.isDebugEnabled()) {
+            log.debug("getAllScopesOfApplicationSubscription : " + stringify(result));
+        }
+        return {
+            error:false,
+            result:result
+        };
+    } catch (e) {
+        log.error(e);
+        return {
+            error:true,
+            result:e
```

Comment:
might cause security concern described at [1] if we have not handled properly at the block layer
[1] https://github.com/wso2/product-apim/issues/3788

Reply:
Fixed with a61e8a6c507360351ecc5afaefe315144054c52e